### DISCUSSION TITLE
How to add results to the quickfix list like Telescope.

### DISCUSSION THREAD

**Discussion by asilvadesigns** [ibhagwan/fzf-lua#1537] [Q&A] [answered]
Hello, 

I'm a new user to this plugin, so please forgive my ignorance I'm sure I've overlooked this detail while scanning through the documentation. In Telescope it seems that for any result set returned from a picker, I'm able to "ctrl-q" or whatever hotkey the user has defined, and it will send those results to the quickfix list. Sometimes these are not intuitive, you can't hop to a highlight for example, but it is extremely useful.

Is there a way to do this with fzf-lua? I'd like to be able to `FzfLua highlights` populate some results and `ctrl-q` to the quickfix list. Thanks!

**Comment by ibhagwan** [comment]
Use `F1` to see the default binds, alt-q is the default send selection to qf, highlights do not support sending to qf as the results aren’t file like entries.

**Reply by asilvadesigns** [reply]
got it, so if I wanted to extract those results, would I use a custom function or selection logic to transform the results?

**Reply by ibhagwan** [reply, marked as answer]
> got it, so if I wanted to extract those results, would I use a custom function or selection logic to transform the results?

Correct, read the wiki on how to create a custom key bind and use the “selected” table to parse the results.

**Reply by asilvadesigns** [reply]
Awesome, thanks so much!
